### PR TITLE
Add username and public key for katstevens

### PR DESCRIPTION
**Pull request by katstevens**
I'm new to GOV.UK 👋 Adding my manifest as per the setup guide.

### CODE REVIEW

File: modules/users/manifests/katstevens.pp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Creates the katstevens user
+class users::katstevens{ govuk_user { 'katstevens':
+    fullname => 'Kat Stevens',
```
Comment (barrucadu) [inline modules/users/manifests/katstevens.pp]: I think this needs to match your email address, so `katherinestevens`

Comment (barrucadu) [reply modules/users/manifests/katstevens.pp]: Whoops, on the line above

Comment (AlanGabbianelli) [reply modules/users/manifests/katstevens.pp]: Yes:
```
Failures:

  1) users::katstevens should have a username of the correct form
     Failure/Error: expect(uname_from_full).to eq(uname_from_email), "name in fullname(#{user[:fullname]}) must match name in email(#{user[:email]})"
       name in fullname(Kat Stevens) must match name in email(katherine.stevens@digital.cabinet-office.gov.uk)
     # ./modules/users/spec/classes/users_spec.rb:39:in `block (3 levels) in <top (required)>'
```

Comment (jonodrew) [reply modules/users/manifests/katstevens.pp]: Yup - see the error message in the build:
```
Failures:

  1) users::katstevens should have a username of the correct form
     Failure/Error: expect(uname_from_full).to eq(uname_from_email), "name in fullname(#{user[:fullname]}) must match name in email(#{user[:email]})"
       name in fullname(Kat Stevens) must match name in email(katherine.stevens@digital.cabinet-office.gov.uk)
     # ./modules/users/spec/classes/users_spec.rb:39:in `block (3 levels) in <top (required)>'
```

Comment (katstevens) [reply modules/users/manifests/katstevens.pp]: Thanks all - pushed a fix. I've changed to `katherine` everywhere for consistency 😸

**Pull request merged**